CHANGELOG.asciidoc

```diff
@@ -314,6 +314,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Run Kafka integration tests on version 1.1.0 {pull}7616[7616]
 - Release raid and socket metricset from system module as GA. {pull}7658[7658]
 - Release elasticsearch module and all its metricsets as beta. {pull}7662[7662]
+- Release munin and treafik module as beta. {pull}7660[7660]
```

Comment:
Ugh, so sorry that I didn't catch this earlier but there is a typo here. The correct spelling is `traefik` (a before the e).